lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -419,6 +456,64 @@
     cluster: mycluster
     id: 1111-1111-1111-1111
 
+- name: Register VM with vnic profile mappings and reassign bad macs
+  ovirt_vms:
+    state: registered
+    storage_domain: mystorage
+    cluster: mycluster
+    id: 1111-1111-1111-1111
+    vnic_profile_mappings:
+      - source_network_name: mynetwork
+        source_profile_name: mynetwork
+        target_profile_name: target_network
+      - source_network_name: mynetwork2
+        source_profile_name: mynetwork2
+        target_profile_name: target_network2
+    reassign_bad_macs: "True"
+
+- name: Register VM with mappings
+  ovirt_vms:
+    state: registered
+    storage_domain: mystorage
+    cluster: mycluster
+    id: 1111-1111-1111-1111
+    role_mappings:
```

Comment:
Shouldn't those examples look like:

```yaml
role_mappings:
 - source:
      name: roleA
    dest:
      name: roleB
```

Wouldn't it be better if it will look like:

```yaml
role_mappings:
   - source_name: roleA
     dest_name: roleB
```

like `vnic_profile_mappings` has.

Reply:
Yes, you are right, it should be like this : 
      - source:
          - name: test
        dest:
          - name: cluster44

I prefer not to restrict mappings to be only for name, I know that right now it is only for name, but using source and dest as objects might be more easy to add any future attributes